ports/psoc6/modules/machine/modmachine.c

```diff
@@ -1,46 +1,24 @@
-
-#include <stdio.h>
-
-
-#include "py/gc.h"
```

Comment:
See my comment in the corresponding header file.

Reply:
Resolved above.